google-cloud/pom.xml

```diff
@@ -92,7 +92,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>google-cloud-vision</artifactId>
-      <version>${project.version}</version>
```

Comment:
My mistake, I didn't look at the context to see that this was for vision. This is correct.